--- keytalkclient-5.5.1/keytalk_ca_updater.sh.orig	2019-04-16 19:51:39.937601019 +0200
+++ keytalkclient-5.5.1/keytalk_ca_updater.sh	2019-04-16 19:53:36.772599403 +0200
@@ -12,8 +12,8 @@
 {
     echo "Start monitoring the System Certificate Trust Store for KeyTak CA changes"
 
-    # Debian/Ubuntu
-    if [ -f /etc/debian_version ]; then
+    # Debian/Ubuntu/Gentoo
+    if [ -f /etc/debian_version ] || [ -f /etc/gentoo-release ]; then
         inotifywait -q -m -e modify,delete,move /usr/local/share/ca-certificates/ | while read dirname event filename
         do
             if [[ "$filename" =~ ^keytalk_.*\.crt$ ]]; then
