--- keytalkclient-5.5.1/etc_systemd_system_keytalk-ca-updater.service.orig	2019-04-17 05:20:46.799128827 +0200
+++ keytalkclient-5.5.1/etc_systemd_system_keytalk-ca-updater.service	2019-04-17 05:19:15.430130090 +0200
@@ -8,7 +8,7 @@
 Restart=always
 RestartSec=1
 User=root
-ExecStart=/usr/local/bin/keytalk/keytalk_ca_updater.sh
+ExecStart=/usr/bin/keytalk_ca_updater.sh
 
 [Install]
 WantedBy=multi-user.target
