--- lib/rdig/content_extractors/hpricot.rb.orig	2008-02-23 17:47:51.000000000 +0100
+++ lib/rdig/content_extractors/hpricot.rb	2008-12-28 16:23:50.000000000 +0100
@@ -47,13 +47,27 @@
         return ''
       end
 
-      # extracts the href attributes of all a tags, except 
+      # extracts the href attributes of all a, area and frame tags, except 
       # internal links like <a href="#top">
       def extract_links(doc)
-        (doc/'a').map { |link|
+        array_a = (doc/'a').map { |link|
           href = link['href']
           CGI.unescapeHTML(href) if href && href !~ /^#/
-        }.compact
+        }
+
+        array_area = (doc/'area').map { |link|
+          href = link['href']
+          CGI.unescapeHTML(href) if href && href !~ /^#/
+        }
+
+        array_frame = (doc/'frame').map { |link|
+          href = link['src']
+          CGI.unescapeHTML(href) if href && href !~ /^#/
+        }
+
+	array_a.concat( array_area.concat( array_frame ) ).compact
+	#total = array_a + array_area + array_frame
+	#total.compact
       end
 
       # Extracts the title from the given html tree
