--- lib/rdig/url_filters.rb.orig	2008-02-24 09:15:15.000000000 +0100
+++ lib/rdig/url_filters.rb	2008-12-28 16:23:23.000000000 +0100
@@ -163,10 +163,11 @@
       return document
     end
 
-    # expands both href="/path/xyz.html" and href="affe.html"
+    # expands href="/path/xyz.html", href="affe.html" and href="../lala.html"
     # to full urls
     def UrlFilters.fix_relative_uri(document)
       #return nil unless document.uri.scheme.nil? || document.uri.scheme =~ /^https?/i
+
       ref = document.referring_uri
       return document unless ref
       uri = document.uri
@@ -174,12 +175,16 @@
       uri.host = ref.host unless uri.host
       uri.port = ref.port unless uri.port || ref.port==ref.default_port
       uri.path = ref.path unless uri.path
-      
-      if uri.path !~ /^\//
+      old_uri_path = uri.path
+
+      if uri.path !~ /^\// || uri.path =~ /^\.\./
         ref_path = ref.path || '/'
         ref_path << '/' if ref_path.empty?
         uri.path = ref_path[0..ref_path.rindex('/')] + uri.path
       end 
+
+      uri.path = uri.path.sub( /\/[^\/]*\/\.\./, "" ) if old_uri_path =~ /^\.\./
+
       return document
     rescue
       p document
