--- src/Image.cpp	2011-02-18 22:18:10.000000000 +0100
+++ src/Image.cpp.new	2016-08-14 20:11:14.481045782 +0200
@@ -2,6 +2,7 @@
 #include <string.h>
 #include <sys/mman.h>
 #include <algorithm>
+#include <unistd.h>
 
 #include "FCam/Image.h"
 #include "FCam/Time.h"
@@ -74,7 +75,7 @@
         flags = MAP_PRIVATE;
     }
     // Make starting offset a multiple of page size, and determine relative offset to true buffer start
-    int pageSize = getpagesize();
+    int pageSize = sysconf(_SC_PAGESIZE);
     int startOfMap = (offset/pageSize)*pageSize;
     int mapOffset = offset-startOfMap;
     // Make mapping size a multiple of page size, rounding up
