--- libpd-0.10.0/Makefile.orig	2016-11-21 18:33:00.531692175 +0100
+++ libpd-0.10.0/Makefile	2016-11-21 18:33:32.342689940 +0100
@@ -200,6 +200,7 @@
 	mkdir -p $(prefix)/include/libpd/util
 	install libpd_wrapper/util/z_print_util.h $(prefix)/include/libpd/util
 	install libpd_wrapper/util/z_queued.h $(prefix)/include/libpd/util
+	mkdir -p $(prefix)/lib
 	install $(LIBPD) $(prefix)/lib
 	if [ -e $(PDCPP) ]; then \
 		install cpp/*.hpp $(prefix)/include/libpd; \
