--- libpd-0.10.0/Makefile.orig	2016-11-25 21:19:17.380710662 +0100
+++ libpd-0.10.0/Makefile	2016-11-25 21:22:18.706697923 +0100
@@ -156,7 +156,7 @@
 libpd: $(LIBPD)
 
 $(LIBPD): ${PD_FILES:.c=.o} ${UTIL_FILES:.c=.o} ${EXTRA_FILES:.c=.o}
-	$(CC) -o $(LIBPD) $^ $(LDFLAGS) -lm -lpthread 
+	$(CC) -o $(LIBPD) $^ $(LDFLAGS),-soname,libpd.$(SOLIB_EXT) -lm -lpthread 
 
 javalib: $(JNIH_FILE) $(PDJAVA_JAR)
 
@@ -182,7 +182,7 @@
 
 # build with LIBPD_UTILS since cpp wrapper uses the ringbuffer
 $(PDCPP): ${PD_FILES:.c=.o} ${LIBPD_UTILS:.c=.o} ${EXTRA_FILES:.c=.o} ${CPP_FILES:.cpp=.o} 
-	g++ -o $(PDCPP) $^ $(CPP_LDFLAGS) -lm -lpthread
+	g++ -o $(PDCPP) $^ $(CPP_LDFLAGS),-soname,libpdcpp.$(SOLIB_EXT) -lm -lpthread
 
 clean:
 	rm -f ${PD_FILES:.c=.o} ${PD_EXTRA_OBJS} ${CPP_FILES:.cpp=.o} ${JNI_FILE:.c=.o}
