--- evelink-0.7.5/setup.py.orig	2015-02-24 07:21:46.818630547 +0100
+++ evelink-0.7.5/setup.py	2015-02-24 07:24:23.540641477 +0100
@@ -11,6 +11,12 @@
 except IOError:
     __readme_contents = ""
 
+
+__docdir = ""
+if os.path.exists("/usr/share/doc"):
+    __docdir = "/usr/share/doc/evelink-" + __version__
+
+
 setup(
     name="EVELink",
     version=__version__,
@@ -31,7 +37,7 @@
         "evelink.thirdparty",
     ],
     data_files=[
-        ('', ['README.md', 'LICENSE']),
+        (__docdir, ['README.md', 'LICENSE']),
     ],
     scripts=["bin/evelink"],
     provides=["evelink"],
