--- pure-data-0.47-1/configure.ac.orig	2017-01-03 16:11:25.861175098 +0100
+++ pure-data-0.47-1/configure.ac	2017-01-03 16:22:10.874196598 +0100
@@ -114,6 +114,7 @@
 AC_SUBST(EXTERNAL_LDFLAGS)
 AC_SUBST([ALSA_LIBS])
 AC_SUBST([JACK_LIBS])
+AC_SUBST([PORTMIDI_LIBS])
 
 dnl Checks for libraries.
 AC_CHECK_LIB([dl], [dlopen])
@@ -211,10 +212,10 @@
 	[AS_HELP_STRING([--enable-portmidi],
 		[use portmidi])],
     [portmidi=$enableval])
-AM_CONDITIONAL(PORTMIDI, test x$portmidi = xyes)
 AS_IF([test x$portmidi = xyes], [
- AC_MSG_NOTICE([using included portmidi])
-])
+    AC_CHECK_LIB(portmidi,pm_init,PORTMIDI_LIBS="-lportmidi",portmidi=no)])
+AM_CONDITIONAL(PORTMIDI, test x$portmidi = xyes)
+
 
 dnl fftw v3
 AC_ARG_ENABLE([fftw],
