--- xooky-nabox-9999/src/main.cpp.orig	2012-10-25 21:27:40.000000000 +0200
+++ xooky-nabox-9999/src/main.cpp	2012-10-25 21:29:20.000000000 +0200
@@ -123,13 +123,9 @@
 // = INITIALIZE LIB PD =
 // =====================
 void initLibPd(){
-	 // The Jack server only seems to run fine at 256 samples per frame and libpd only processess samples
-	 // in chunks (ticks) of n*64 samples at a time. We need to set the ticksPerBuffer to 4.
-
-
 	 // init the pd engine
 	 libpd_init();
-	 libpd_init_audio(2, 2, sampleRate, 4); //nInputs, nOutputs, sampleRate, ticksPerBuffer
+	 libpd_init_audio(2, 2, sampleRate ); //nInputs, nOutputs, sampleRate
 	 
 	 // send compute audio 1 message to pd
 	 libpd_start_message(1);
@@ -196,7 +192,10 @@
 	 }
  
 	 // DSP Magic ;)
-	 libpd_process_float(input, output);
+	 // The Jack server only seems to run fine at 256 samples per frame and libpd only processess samples
+	 // in chunks (ticks) of n*64 samples at a time. We need to set the ticksPerBuffer to 4.
+
+	 libpd_process_float(4, input, output);
 	 
 	 for(unsigned int i=0; i<nframes; i++){
 			*out1 = output[i*2];
